spring-cloud-azure-samples/eventhubs-binder-sample/pom.xml

```diff
@@ -23,4 +23,15 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
Why add this in sample?

Reply:
User can use this plugin to deply jar to app service.